docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -118,16 +130,32 @@ While app schemes are generally easier to set up, Universal URLs and App Links a
   </mark>
 </p>
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 Calling the token endpoint is the second step in the authorization flow and usually happens in your backend if your application's Client Type is `Confidential` <mark>What if the Client Type is **Public**?</mark>. Make sure to a call the authorization endpoint first to retrieve an authorization code.
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 <p>
   <mark>
     Are they parsing the <code>redirectUrl</code> returned when calling the function <code>Embedded.authenticate</code> for a code in the query parameters and then exchanging that code for an access token?
   </mark>
 </p>
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 If your application is using the [NextAuth](https://next-auth.js.org) provider (see the Javascript Authorization example using Automatic Invocation Type) <mark>Where is this example? We tell them to refer to it but we don't give them a link. Is it in the SDK?</mark>, you won't need to complete authentication with a token exchange.
```

Comment:
I think this is just the example included [here](includes/_authentication-via-automatic.mdx), which is already included in this document. It appears if Javascript and Automatic are selected. Maybe this should be included here too if Manual is selected. @turnipdabeets, what do you think?

Reply:
I think when the doc was rearranged it might have lost this link. I think this code snippet was only for Automatic.